src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,127 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table = new Node[INITIAL_CAPACITY];
```

Comment:
The field `table` should be initialized in the constructor. It's a good practice to initialize instance variables in the constructor or with a default value at the time of declaration. This makes the code more readable and maintainable.